analyzers/tests/SonarAnalyzer.UnitTest/TestCases/DoNotCheckZeroSizeCollection.CSharp9.cs

```diff
@@ -12,14 +13,18 @@
 {
 }
 
+if (someEnumerable.Count() is  >= localConst_Zero) // Noncompliant
+{
+}
+
 if (((someEnumerable.Count()), anotherEnumerable.Count()) is (>= 0, // Noncompliant
 //                                                            ^^^^
     >= 0)) // Noncompliant
 //  ^^^^
 {
 }
 
-if (someEnumerable.Count() is < 0) { } // FN
+if (someEnumerable.Count() is < 0) { } // Noncompliant
```

Comment:
Let's add one example with localConst_Zero here. It's nice to show that we support it as well.